docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -170,6 +171,14 @@ similar to `Environment`.
 
 This key can be listed multiple times.
 
+### `AutoUpdate=`
+
+Indicates whether the container will be auto-updated (`[podman-auto-update(1)](podman-auto-update.1.md)`). The following values are supported:
```

Comment:
also here, you need to remove the code block, the markdown link will not work inside it